### ISSUE TITLE
[Adaptive Card iOS - AdaptiveCard.json]: Luminosity ratio for placeholder text 'Add a comment' with respect to background is '1.723.1' which is less than 4.5:1

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#167]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 17.5.1
Application: Adaptive Cards
App Version: Version 1.0 (2.4.0-beta.24.5.30.1)
Tool: Accessibility Insight For Window

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.0" control and select it.
4. Navigate to 'Elements' control and activate it.
5. Navigate to 'AdaptiveCards.json' under 'Elements' list item and activate it.
6. Card will be appear.
7. Navigate to 'Comment' button and activate it.
8. Navigate to 'Add a comment' text field.
9. Open the tool, check the contrast ratio, and observe the issue.

**Actual Result:**
Luminosity ratio for placeholder text 'Add a comment' is '1.723.1' with respect to background, which is less than 4.5:1.

**Expected Result:**
Luminosity ratio for placeholder text 'Add a comment' with respect to background should be equal to or greater than 4.5:1

**Notes:**
Issue is repro throughout the application with placeholder text like this.

**User Experience:**
When a minimum contrast ratio is not provided between text and its background, it is difficult for low-vision users as well as anyone in extreme lighting environments to read the content.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

### Screenshots

![Luminosity ratio for the Enter your comment placeholder text is less than 4 5 1](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/97d7adfd-8402-43f4-bad0-0b19433060ab)

### Card JSON

```JSON
NA
```

**Comment by GabrielMedAlv** [comment]
Fix PR for this in https://github.com/microsoft/Teams-AdaptiveCards-Mobile/pull/244

**Comment by vagpt** [comment]
Currently on build 1.0 (2.4.0-beta.24.11.3.1) there is no placeholder text is present, hence closing this issue.

![Image](https://github.com/user-attachments/assets/40de96db-977e-40fc-8f9d-0ddd8d4ad8cd)
